docs/Gemfile.lock

```diff
@@ -401,7 +401,7 @@ GEM
       io-console (~> 0.5)
     rexml (3.2.8)
       strscan (>= 3.0.9)
-    rouge (3.26.0)
+    rogue (3.26.0)
```

Comment:
Thanks, however, this package came must remain the same.

Reply:
Oh! Sorry my bad...

Reply:
Updated the corrections